deploy/role_bindings.yaml

```diff
@@ -0,0 +1,60 @@
+# allow the operator to watch configmaps in openshift-config
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: osd-metrics-exporter-watch-configmaps
+  namespace: openshift-config
```

Comment:
I don't believe OLM will apply this because its in a different namespace than the operator

Reply:
How is that working with the Cloud-ingress-operator? The cio is creating a rolebinding in the kube-system namespace ( see https://github.com/openshift/cloud-ingress-operator/blob/master/deploy/20_cluster_config_v1_reader_role_binding.yaml ). Would I need to add the same owner labels, like:
```
  labels:
    owner: osd-metrics-operator
    owner.namespace: openshift-osd-metrics
```
?

Reply:
Example: 
Role https://github.com/openshift/cloud-ingress-operator/blob/master/resources/20_cloud-ingress-operator_kube-apiserver.Role.yaml
Rolebinding: https://github.com/openshift/cloud-ingress-operator/blob/master/resources/20_cloud-ingress-operator_kube-apiserver.RoleBinding.yaml

Deployment of role/binding: https://github.com/openshift/cloud-ingress-operator/blob/master/hack/olm-registry/olm-artifacts-template.yaml#L258-L309